Cargo.toml

```
@@ -27,7 +27,7 @@ vendored = ["anoncreds-clsignatures/openssl_vendored"]
 w3c = ["base64", "chrono", "rmp-serde"]
 
 [dependencies]
-anoncreds-clsignatures = "0.3.0"
+anoncreds-clsignatures = { git = "https://github.com/DSRCorporation/anoncreds-clsignatures-rs.git", branch = "feat/expose-proof-predicates" }
```

0.3.1 has been released

```suggestion
anoncreds-clsignatures = "0.3.1"
```